src/ioctls/vcpu.rs

```diff
@@ -1049,6 +1049,31 @@ impl VcpuFd {
         Ok(reg_value)
     }
 
+    /// Notify the guest about the vCPU being paused.
+    ///
+    /// This signals to the host kernel that the specified guest is being paused by
```

Comment:
Hmm, isn't this a signal from the host to the guest kernel?

Reply:
Seems it means "signal host to notify the guest":)

Reply:
Yes this comes directly from the KVM API documentation. Basically you invoke this ioctl to let your KVM host know that it should set a flag which will be later read by the guest.